doctor_visits/delphi_doctor_visits/patch.py

```diff
@@ -0,0 +1,57 @@
+"""
+This module is used for patching data in the delphi_doctor_visits package.
+
+To use this module, you need to specify the range of issue dates in params.json.
+
+It will generate data for that range of issue dates, and store them in batch issue format:
+[name-of-patch]/issue_[issue-date]/doctor-visits/actual_data_file.csv
```

Comment:
suggestion: add example output

```suggestion
[name-of-patch]/issue_[issue-date]/doctor-visits/actual_data_file.csv

The run using the above example `params.json` will create a directory 'covidcast-indicators/doctor_visits/AprilPatch' containing:

AprilPatch/
    issue_20240420/
        doctor-visits/
            20240408_state_smoothed_adj_cli.csv
            20240408_state_smoothed_cli.csv
            ...
    issue_20240421/
        doctor-visits/
            20240409_state_smoothed_adj_cli.csv
            20240408_state_smoothed_cli.csv
            ...
```